### DISCUSSION TITLE
Level page

### DISCUSSION THREAD

**Discussion by sunehs** [elmadev/elmaonline-site#419] [Feature discussion]
For general discussion of the level page: https://elma.online/levels/2

This post will be updated to contain planned features and changes as well as ideas that needs to be fledged out. Discuss them below. If you want to work on one of these, create an issue if it doesn't exist and assign yourself.

**Planned**

- [x] Replay tags: Filter the replay list using tags instead of checkboxes

**Ideas**
- All times: Maybe some filter or pagination so you can see more than top 10k

**Comment by j-masci** [comment]
- [x] show level pack(s) that level belongs to on level page.

**Comment by j-masci** [comment]
Using new level stats data (ie. https://elma.online/levels/2)
- Show some aggregated level stats data (attempts, time, win/death/esc %, leader count, unique leaders, or whatever makes sense). Could do some silly ones if they are not very prominent: left/right/alo volt ratios. Top speed ever driven / finished with.
- number of kuskis that played / finished the level.
- Highlight the top battle time in the top 10 times if possible (unless this doesn't even require levelstats, idk). Sometimes it's nice to know whether a level was beaten after a battle was run without having to click the battle and compare times.

**Comment by j-masci** [comment]
Using new level stats data, on Level Search page (https://elma.online/search?q=asdf&t=level)
- Show some pretty basic stats (KuskiCountF, LeaderCount, Top time, perhaps). For levels with duplicate names it would be nice to see how many other players played it.
